improved_mi_interpret_results.py

```diff
@@ -82,7 +84,7 @@ def get_pred_mem_mi(per_instance_loss, proposed_mi_outputs, method=1, fpr_thresh
 			#plt.plot(list(range(1, 101)), list(map(lambda x: -x, threshs)))
 			#plt.show()
 			return max(0, threshs[0]), pred_membership
-		else:
+		else:  # In this case, run the Merlin attack.
```

Comment:
I think the comment on line 87 should come at line 90 (where the outer 'else' is). The inner 'if-else' case is for using per-class threshold. Outer 'if-else' decides between using yeom or merlin.